app/containers/BalanceContainer/index.js

```diff
@@ -0,0 +1,78 @@
+/*
+ *
+ * BalanceContainer
+ *
+ */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import Helmet from 'react-helmet';
+import selectBalanceContainer from './selectors';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+import styles from './styles.css';
+import BalanceCell from 'components/BalanceCell';
+import $ from 'jquery'; 
+import BigNumber from 'bignumber.js';
+
+export class BalanceContainer extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  constructor(props) {    
+    super(props);
+
+    this.state = {
+      balance: 0,
+      name: ""
+    };
+  }
+
+  componentDidMount() {
+    if (this.props.token == null) {
+      return;
+    }
+
+    this.setState({name: this.props.token.symbol});
+
+    var data = $.param(this.props.token.balanceCallData());
+    var serverUrl = "https://rpc.myetherwallet.com:8443/api.mew";
+    var parentObj = this;
+    $.ajax({
```

Comment:
`ajax` is not necessary here. Use the [Fetch API](https://davidwalsh.name/fetch).